.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 7.4
+          php-version: 8.3
```

Comment:
It's best to make the base test the lowest supported version PHP supported by the composer.lock file, e.g. 8.2.